# Filter status

std_msgs/Header header

# Quaternion status, 0 = invalid, 1 = valid, 2 = magnetically referenced
uint16 quatStatus

# Bias status, 0 = invalid, 1 = valid
uint16 biasStatus
